sdks/java/harness/src/main/java/org/apache/beam/fn/harness/FnApiDoFnRunner.java

```diff
@@ -1175,6 +1183,12 @@ private Progress getProgressFromWindowObservingTruncate(double elementCompleted)
 
   @VisibleForTesting
   static Progress scaleProgress(Progress progress, int currentWindowIndex, int stopWindowIndex) {
```

Comment:
`Progress SplittableFnDataReceiver.getProgress()` should check to see if the current window is null just like `getProgressFromWindowObservingTruncate` does.

Reply:
I thought hte check on tracker existence was sufficient but added window check too.